src/main/java/Lesson6/DivisibilityBy3.java

```diff
@@ -0,0 +1,24 @@
+package Lesson6;
+
+public class DivisibilityBy3 {
+    public static void main(String[] args) {
+        validate(new int[]{10, 11, 20, 30});
+    }
+
+    public static void validate(int[] values) {
+        int i;
+        for (i = 0; i < 4; i++) {
+            try {
+                if (values[i] % 3 != 0) {
+                    throw new IllegalArgumentException("Liczba:" + values[i] + " Nie jest podzielna przez 3");
+
+                } else System.out.println("Liczba:" + values[i] + " Jest podzielna przez 3");
```

Comment:
Wiem, że język na to pozwala, ale używaj klamr else { ... } , jest o wiele czytelniej ;)
Jedyne kiedy się klamr nie używa to w przypadku tzw operatora Elvisa np:
```
boolean variable = values[i] % 3 != 0 ? true : false;
```